sdk/src/client/bindings/python/iota_client/client.py

```diff
@@ -382,4 +526,175 @@ def as_dict(self):
+    StorageDepositReturn = 1
+    Timelock = 2
+    Expiration = 3
+    StateControllerAddress = 4
+    GovernorAddress = 5
+    ImmutableAliasAddress = 6
+
+class UnlockCondition():
+    def __init__(self, type=None, address=None, amount=None, unix_time=None, return_address=None) -> None:
+        """Initialize an UnlockCondition
+        
+        Parameters
+        ----------
+        type : UnlockConditionType
+            The type of unlock condition
+        address : Address
+            Address for unlock condition
+        amount : int
+            Amount for storage deposit unlock condition
+        unix_time : int
+            Unix timestamp for timelock and expiration unlock condition
+        return_address : Address
+            Return address for expiration and storage deposit unlock condition
+        """
+        self.type = type
+        self.address = address
+        self.amount = amount
+        self.unix_time = unix_time
+        self.return_address = return_address
+
+    def as_dict(self):
+        config = {k: v for k, v in self.__dict__.items() if v != None}
+        
+        if 'type' in config:
+            config['type'] = config['type'].value
+
+        if 'address' in config:
+            config['address'] = config['address'].as_dict()
+
+        if 'return_address' in config:
+            config['return_address'] = config['return_address'].as_dict()
+
+        if 'amount' in config:
+            config['amount'] = str(config['amount'])
+
+        return config
+
+class AddressType(Enum):
+    ED25519 = 0
+    ALIAS = 8
+    NFT = 16
+
+class Address():
+    def __init__(self, type, address_or_id):
+        """Initialize an Address
+        
+        Parameters
+        ----------
+        type : AddressType
+            The type of the Address
+        address_or_id : string
+            The address to use. Can either be an hex encoded ED25519 address or NFT/Alias id
+        """
+        self.type = type
+        self.address_or_id = address_or_id
+
+    def as_dict(self):
+        config = dict(self.__dict__)
+
+        config['type'] = config['type'].value
+        
+        if self.type == AddressType.ED25519:
+            config['pubKeyHash'] = config.pop('address_or_id')
+        elif self.type == AddressType.ALIAS:
+            config['alias_id'] = config.pop('address_or_id')
+        elif self.type == AddressType.NFT:
+            config['nft_id'] = config.pop('address_or_id')
+
+        return config
+
+class FeatureType(Enum):
+    Sender=0
+    Issuer=1
+    Metadata=2
+    Tag=3
+
+class Feature():
+    def __init__(self, type, sender=None, issuer=None, data=None, tag=None):
+        """Initialise a feature
+
+        Parameters
+        ----------
+        type : FeatureType
+            The type of feature
+        sender : Address
+            Sender address
+        issuer : Address
+            Issuer Address
+        data : string
+            Hex encoded metadata
+        tag : string
```

Comment:
Is this missing a description?

Reply:
Guess I just was too lazy to write `tag of the output` or something again 😆 
Maybe I could add something like `indexable tag of output` to better describe what it could/should be used for? 🤔

Reply:
Also just say that it needs to be hex encoded as above with the metadata